contact-duplicate/src/app/app.functions/duplicateContact.js

```diff
@@ -88,11 +89,11 @@ const setAssociations = (token, contact, associations) => {
     );
   }
 
-  return Promise.all(associationsPromises).then(() => contact);
+  return Promise.all(associationsPromises);
```

Comment:
I don't think it matters, but for consistency, do we want to `return await Promise.all(...)`?